framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -180,6 +180,18 @@ public int runTests() throws Exception {
   	  queryMemoryUsage();
     }
 
+    // Run Apache Minio server if needed for the S3 storage
+    if(cmdParam.excludeDependencies == null || !cmdParam.excludeDependencies.contains("s3")) {
+      Utils.startMinio();
+    } else {
+      // Disable S3 storage plugin if Minio server is down
```

Comment:
I think we should rename `s3` plugin as `s3test` or `minio` or something similar. If anyone has configured the S3 plugin to point to Amazon's S3, then we will disable or modify the plugin unintentionally.

Reply:
Done. Renamed "s3" to "s3minio".